index.bs

```diff
@@ -1803,14 +1862,15 @@ To <dfn noexport>parse source-registration JSON</dfn> given a [=byte sequence=]
 1. If |aggregationKeys| is null, return null.
 1. Let |triggerDataCardinality| be the user agent's
     [=navigation-source trigger data cardinality=].
-1. Let |maxAttributionsPerSource| be the user agent's
-    [=max attributions per navigation source=].
 1. If |sourceType| is "<code>[=source type/event=]</code>":
     1. Round |expiry| away from zero to the nearest day (86400 seconds).
     1. Set |triggerDataCardinality| to the user agent's
         [=event-source trigger data cardinality=].
-    1. Set |maxAttributionsPerSource| to the user agent's
-        [=max attributions per event source=].
+1. Let |maxAttributionsPerSource| be the result of running
+    [=parse an optional 64-bit signed integer=] with |value|, "`max_event_level_reports`", and
```

Comment:
Seems like a previous comment of mine was lost here: This differs from the explainer, which says this field is an integer, not a string: https://github.com/WICG/attribution-reporting-api/blob/main/flexible_event_config.md#api-changes

Reply:
Done.

Reply:
(Treated as an integer)

Reply:
Isn't this still requiring `max_event_level_reports` to be a string?

Reply:
Yes sorry, forgot to push the recent change. I'll upload a combined fix which also re-adds `start_time` today

Reply:
PTAQL